### DISCUSSION TITLE
Change the orientation of a scalar bar's title

### DISCUSSION THREAD

**Discussion by piyueh** [pyvista/pyvista#2623] [Q&A]
## Description

When using a vertical scalar bar, the title is still written in a horizontal way at the top of the scalar bar. Is it possible to make the title vertical at the left or right of the scalar bar?

## Example Data

For example, this is what a vertical bar looks like in `pyvista`: 

![image](https://user-images.githubusercontent.com/7220966/134591622-fc61d23d-d009-4c9e-bd1f-8888f1e99398.png)

What I'm trying to achieve is something like this:

![image](https://user-images.githubusercontent.com/7220966/134591771-dabeea9a-439f-43bf-8e53-2057cb828eb3.png)

Thanks in advance!

**Comment by avaruuser** [comment]
I am also struggling with this. Are there any solutions for this issue?

**Comment by MatthewFlamm** [comment]
Unfortunately, I do not know of a way to accomplish this today.  One thought is that if `Plotter.add_text` supported text rotation, then the user could add in a custom title.  This would be a nice feature request.

**Reply by MatthewFlamm** [reply]
See #2794
